src/Events/Dispatcher.php

```diff
@@ -30,7 +30,7 @@ class Dispatcher extends BaseDispatcher
      * @param  int  $priority
      * @return void
      */
-    public function listen($events, $listener, $priority = 0)
+    public function listen($events, $listener = null, $priority = 0)
```

Comment:
We'll need to implement / make sure our dispatcher supports the changes made in https://github.com/laravel/framework/commit/46084d946cdcd1ae1f32fc87a4f1cc9e3a5bccf6

Reply:
Done, added a unit test for the dispatcher that tests all 3 types of event listeners it should accept.
I have no clue how to trigger a QueueClosure, as it requires a massive part of the framework to be booted up and I don't know how to get it working so the event bus works. I settled for testing that it accepts the argument without fault.